spec/common/password_protected_correct_password_spec.rb

```diff
@@ -0,0 +1,22 @@
+# frozen_string_literal: true
```

Comment:
I think all specs (since now it's 3 of them_ should be placed in `password_protected` folder

and all `docx` files from this folder should be moved to something like `spec/common/password_protected/files`